web_monitoring/differs.py

```diff
@@ -51,11 +51,20 @@ def pagefreezer(a_url, b_url):
     # Just send PF the urls, not the whole body.
     # It is still useful that we downloaded the body because we are able to
     # validate it against the expected hash.
-    return web_monitoring.pagefreezer.compare(a_url, b_url)
+    return web_monitoring.pagefreezer.PageFreezer(a_url, b_url)
 
+def compute_dmp_diff(a_text, b_text, timelimit=4):
+    diff_codes = {'=': 0, '-': -1, '+': 1}
```

Comment:
This is a really minor optimization, but it might make sense to pull this definition out of the function so you only allocate the dictionary once.